client/src/Components/Post/Post.js

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.css';
+import './styles.css';
```

Comment:
Isn't it enough just to import it in App.js? I don't remember well. Could you please double-check?

Reply:
I thought that css imported here was local. I guess it need to make it into a module